src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3418,6 +3419,25 @@ bool MusicXMLParserDirection::isLikelyTempoText(const track_idx_t track) const
     return false;
 }
 
+void MusicXMLParserDirection::handleFraction()
+{
+    if (!configuration()->inferTextType()) {
+        return;
+    }
+
+    String rawWordsText = m_wordsText;
+    static const std::regex re("(<.*?>)");
+    rawWordsText.remove(re);
+    rawWordsText = rawWordsText.simplified();
+
+    static const std::regex frac("^([0-9])(/)([0-9])$");
```

Comment:
Fraction 12/8 is never possible here? I ask because the regex only works with one digit

Reply:
It's not expected. This has a very specific use case